resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -225,7 +227,7 @@ def main(args):
             infos.append("1000G")
 
         if args.no_control:
-            if help["ExAC_COL_VALID"] and any(af > 0.001 for af in map(float, extract_info(help["ExAC_COL"], "AF").split(','))):
+            if help["ExAC_COL_VALID"] and any(af > 1.0 for af in map(float, extract_info(help["ExAC_COL"], "AF").split(','))):
```

Comment:
This is a selector with very different value? So this is rather a major version bump, right?

Reply:
Similar to the SNV workflow, ExAC is added to the gnomAD WES, so no need for a separated filter from EXAC.